src/tyr/route_serializer_osrm.cc

```diff
@@ -549,6 +549,14 @@ json::ArrayPtr intersections(const valhalla::DirectionsLeg::Maneuver& maneuver,
       edges.emplace_back(((prior_heading + 180) % 360), entry, true, false);
     }
 
+    if (i > 0 && !arrive_maneuver) {
+      bool entry = (arrive_maneuver) ? true : false;
+      for (uint32_t m = 0; m < node->intersecting_edge_size(); m++) {
+        auto intersecting_edge = node->GetIntersectingEdge(m);
+        edges.emplace_back(intersecting_edge->begin_heading(), entry, true, false);
```

Comment:
as these are outgoing edges - boolean flags should be set vice versa, nope?
```suggestion
        edges.emplace_back(intersecting_edge->begin_heading(), entry, false, true);
```